python/cloudtik/providers/aws/defaults.yaml

```diff
@@ -81,6 +81,8 @@ setup_commands:
     - which cloudtik || pip install -U "cloudtik[aws] @ http://23.95.96.95:8000/latest/cloudtik-0.9.0-cp37-cp37m-manylinux2014_x86_64.whl"
     - cloudtik-spark install
 
+bootstrap_commands: []
```

Comment:
This can be added to defaults-shared for once to avoid duplicates.

Reply:
Have updated.